cypress/integration/Case07_Send_money.js

```diff
@@ -15,16 +15,41 @@ describe('Test case 7: Ability to send money', () => {
       LoginPage.mnemonicsInput.type(mnemonic)
       LoginPage.recoverWalletButton.click()
       LoginPage.yayButton.click()
-      HomePage.claimButton.click().invoke('text').then(text => {
-        cy.log(text)
-        if (text == 'Queue') {
-          const urlRequest = Cypress.env('REACT_APP_SERVER_URL')
-          const bodyPass = Cypress.env('CYPRESS_GUNDB_PASSWORD')
-          cy.request('POST', urlRequest + '/admin/queue', [{ password: bodyPass, allow: 1 }])
-        }
+      HomePage.claimButton.click()
+      cy.wait(2000) // wait for a pop-up to appear confirming the decision in the queue
+      cy.get('#root')
```

Comment:
aren't this search logic so complicated ? probably we could go simpler using testid ?

Reply:
since conditional testing in cypress is a bit difficult, regarding the  interaction with the elements depending on their presence or not, I found such a way out - count the number of buttons on the screen and depending on their number perform certain actions.
this applies to the moment of pushing the queue at the claiming

Reply:
but why do not add testid and just wait for testid you're need  appears ?
collecting numbers of buttons is badd approach. any small design change will break everything 
we need more exact solutions for avoid spending a lot of resources for adjusting test after each design change